graylog2-server/src/main/java/org/graylog/plugins/views/search/permissions/SearchUser.java

```diff
@@ -91,6 +92,10 @@ public boolean isAdmin() {
         return this.currentUser.isLocalAdmin() || isPermitted("*");
     }
 
+    public UserStreams streams(PermittedStreams permittedStreams) {
+        return new UserStreams(this, permittedStreams);
```

Comment:
We could also inject `PermittedStreams` and instantiate `UserStreams` in the constructor of `SearchUser` and make this method a plain getter.

Reply:
Good point, Implemented, thanks! :+1: